cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -45,7 +45,10 @@ def recid(self, key, value):
 @model.over("agency_code", "^003")
 def agency_code(self, key, value):
     """Control number identifier."""
-    return value
+    if isinstance(value, str):
```

Comment:
I added this check because some records do have a `datafield` like this `<datafield tag=\"003\" ind1=\" \" ind2=\" \">\n  </datafield>\n` instead of a `controlfield` like `<controlfield tag=\"003\">SzGeCERN</controlfield>\n`.
With the `controlfield` we get the value as a `String` but with the `datafields` we get an empty `GroupableOrderedDict`, this was causing that the record could not be correctly indexed in ES since the dynamic mapping recognized it as a `text` but with the empty `GroupableOrderedDict`, it was trying to index an empty array, something like:
```
agency_code:[
       __order__: [ ]
]
```